get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CHALLENGE_ID=$1
+NODEJS_TEST_REPORT_JSON_FILE="${SCRIPT_CURRENT_DIR}/coverage/coverage-summary.json"
+NODEJS_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/coverage.tdl"
+
+( cd ${SCRIPT_CURRENT_DIR} && npm install && npm run coverage || true 1>&2 )
```

Comment:
Is the coverage computed per file of per package?
Have you tried having multiple files in the package?

Reply:
Good point, it might be per file, I will check this out and fix it for all files in that package. Is there a package concept? Might have to aggregate the numbers for all files in package.

Reply:
I do not think there is such a thing as a package, but we will use folder as a proxy for packages.

Reply:
I thought the same, and that our dir name is acting as package name

Reply:
About https://github.com/julianghionoiu/tdl-runner-nodejs/pull/3/files#r192602210, have we considered checking this for other languages as well?

Reply:
Yes. I think we should improve our testing to cover a scenario with multiple files in package.

Reply:
Done for Nodejs, let's discuss this for other languages plus via the dpnt-coverage repo